src/api/fareContracts.ts

```diff
@@ -1,19 +1,29 @@
 import {AxiosRequestConfig} from 'axios';
 import {getCustomerId} from '../utils/customerId';
 import client from './client';
+import {LanguageAndText} from './utils';
 
-export async function list(): Promise<ListTicketsResponse> {
+export async function listFareContracts(): Promise<FareContract[]> {
   const customerId = await getCustomerId();
 
   const url = 'ticket/v1/ticket/' + customerId;
   const response = await client.get<ListTicketsResponse>(url);
 
+  return response.data.fare_contracts;
+}
+
+export async function listPreassignedFareProducts() {
+  const url =
+    'https://gateway-5pwmu3xz7a-ew.a.run.app/api/v1/reference-data/ATB/preassigned-fare-products';
```

Comment:
Now using temporary endpoints, as is it not yet deployed in the staging environment. Should not release with these endpoints.

Reply:
Updated to the mittatb-endpoints.